README.md

```diff
@@ -79,24 +79,10 @@ Autoloading is PSR-0 friendly.
 
 ### Via [Packagist](https://packagist.org/packages/ronanguilloux/isocodes) & [Composer](http://getcomposer.org/doc/00-intro.md)
 
-Create a composer.json file:
-
-```json
-{
-    "require": {"ronanguilloux/isocodes": "@stable"}
-}
-```
-
-Grab composer:
-
-```bash
-$ curl -s http://getcomposer.org/installer | php
-```
-
-Run install (will build the autoloading):
+Require the latest version of `ronanguilloux/isocodes` with Composer
 
 ```bash
-$ php composer.phar install
+composer require ronanguilloux/isocodes
 ```
 
```

Comment:
So just change that, or do I need to put the 'create composer.json' part back?

As a note, `composer require ronanguilloux/isocodes:~1.1` will have the same result, because it will automatically find the best release, using SemVer, and add that to composer.json.

With the latest composer version, it adds `^1.1` as version constraint.